### DISCUSSION TITLE
Transmission upgrade to v4.

### DISCUSSION THREAD

**Discussion by ozhound** [tteck/Proxmox#1040] [Help Needed] [answered]
Hi, thankyou for these scripts, made my experience with Proxmox so much better.

I have used the transmission script recently, however it installs version 3. Version 4, with many fixes would be my preference. How would I upgrade the transmission server that the script installs to version 4?

thanks!

**Comment by tteck** [comment, marked as answer]
Transmission is installed via the package manager. If a newer version is available, you can update it by running the command 
`apt update && apt upgrade`. It's common for the package manager to be slower in adding the latest versions.

**Reply by Pims8383** [reply]
Hello,

Transmission V4.0.5 is working on  Ubuntu 24.04 LXC (not availalble in debian 11 or 12 neither in Ubuntu 22.04 and 23.04)

Can the LXC image  move from debian 11 to  Ubuntu 24.04 to allow the update of transmission ? 

Thanks

**Reply by burgerga** [reply]
Discussion https://github.com/transmission/transmission/discussions/5585 mentions some contact with Debian maintainers, but since that also was already a while ago I wouldn't hold my breath.

I think moving to Ubuntu could be a solution, in the meantime I followed the official instructions at https://github.com/transmission/transmission/blob/main/docs/Building-Transmission.md#debian-12--bookworm to build it from source. You probably need to increase your boot disk slightly, for me 4GB was a tiny bit too small.

**Comment by ozhound** [comment]
Rightio, makes sense. They must not have pushed it to the repo's yet, as that said there wasnt any upgrades.

thanks!

**Reply by Drallas** [reply]
Unfortunately still not available also not in Bullseye or backports repo's. 😢
